ddtrace/tracer/textmap_test.go

```diff
@@ -401,12 +406,16 @@ func TestTextMapPropagator(t *testing.T) {
 	})
 
 	t.Run("InjectExtract", func(t *testing.T) {
+		// TODO: Why is the 128-bit trace id not being propagated correctly by default?
+		os.Setenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED", "true")
+		defer os.Unsetenv("DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED")
```

Comment:
@katiehockman I found the issue. I commited a change to show that `MaxTagsHeaderLen` was missing in the declaration of the propagator config.

When this field is zero, it doesn't encode any propagating tag. It's stated literally in the code, so I assume it's an intended behavior: https://github.com/DataDog/dd-trace-go/blob/27896461c43b3ce073a519de8b68284dc733dfb5/ddtrace/tracer/textmap.go#L323

I left your TODO comment so you can remove it if the fix is correct.

Reply:
Ah yep you're right. I think we just don't expect people to create Propagators this way usually.